apache/superset
# feat: provide support for an overridable template to be included in every view

### SUMMARY

- introduces a template file (which can be overridden) that is included in all superset views. The template file can be used to insert valid html snippets to be run in superset's frontend.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

N/A
### TEST PLAN

- add a js snippet to superset/templates/tail_js_custom_extra.html, eg `console.log("IT WORKS!")`
- navigate open superset to any view
- console output should contain message
- html should include the snippet
### ADDITIONAL INFORMATION
